src/pattern-library/components/patterns/data/Scroll.js

```diff
@@ -0,0 +1,187 @@
+import { Scroll, ScrollContainer, ScrollContent } from '../../../../next';
+import Library from '../../Library';
+import Next from '../../LibraryNext';
+
+import { SampleListElements } from '../samples';
+
+export default function ScrollPage() {
+  return (
+    <Library.Page
+      title="Scroll components"
+      intro={
+        <>
+          <p>
+            The <code>Scroll</code> family of presentational components provide
+            support for contained, scrollable content:
```

Comment:
It would be good to mention `ScrollBox` here as the recommended combination of these lower-level components.